runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotConfig.java

```diff
@@ -253,11 +253,23 @@ default boolean screenshotHighGamble()
 		return false;
 	}
 
+	@ConfigItem(
+		keyName = "collectionLogEntries",
+		name = "Screenshot collection log entries",
+		description = "Configures whether or not screenshots are automatically taken of new collection log entries",
```

Comment:
```suggestion
		description = "Take a screenshot when completing an entry in the collection log",
```

Reply:
I was on the fence of how to format the description since most use the "configures whether" format, while more recent ones use the "take a screenshot" format. I have no issue going with either and can easily commit the suggested change, but I think for consistency sake there should be a consistency pass on the descriptions in the plugin in a separate PR.

Reply:
Generally I do not like "Configure whether" wording, as it's clear the user is already viewing plugin configuration at the point this would be displayed. I understand this is not consistent across options, but new entries at least should be better-worded.